crates/runtime/src/lib.rs

```diff
@@ -55,6 +67,7 @@ pub struct Builder {
     engine: Engine,
     module: Module,
     state_builder: SlightCtxBuilder,
+    io_redirects: Option<IORedirects>,
```

Comment:
Would it be better to add `WasiCtx` as a field to the Builder so that users can pass any configuration of `WasiCtx` struct to slight runtime?

Reply:
Perhaps, I'm just not clear on how that would look. However, it would be something I'd like to explore in a future PR.

Reply:
Could you please raise an issue to create a backlog item for this? After that, feel free to resolve this convo.